app.rb

```diff
@@ -0,0 +1,104 @@
+require 'sinatra'
+require "sinatra/reloader"
+require_relative 'lib/database_connection'
+require_relative 'lib/user_repository'
+require_relative 'lib/post_repository'
+require 'date'
+
+
+DatabaseConnection.connect
+DatabaseConnection.exec(File.read('./seeds/database_setup.sql'))
+
+class Application < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+    also_reload 'lib/post_repository'
+    also_reload 'lib/user_repository'
+  end 
+
+  enable :sessions
+
+  def check_logged_in
+    if session[:user_id] == nil
+      redirect '/'
+    end
+  end 
+
+  def set_session(username, user_id)
+    session[:username] = username
+    session[:user_id] = user_id
+  end 
+
+  get '/' do
+    return erb(:index)
+  end 
+
+  get '/message_board' do
```

Comment:
do you need to print the session[:user_id]? could be nice to use some whitespace for clarity